### DISCUSSION TITLE
Packet Viewing with parameter

### DISCUSSION THREAD

**Discussion by ugurmelihsurme** [yamcs/yamcs#666] [Q&A]
Hi,
How could we get to tm packages not as Base64 but as a c structure? In other words, the values ​​of the parameters in the package can be displayed together with the package.

Example: I couldn't find a better example :)
tmpacket
{
 int speed; 
 int  x;
 int y;
  int z;
}

tmpacket
 |
 |->speed 45
 |->x 3 
 |->y 4
 |->z 5

**Comment by xpromache** [comment]
where do you want it displayed?

**Comment by ugurmelihsurme** [comment]
Hi,
Actually, I would like to view it on packet display if it is possible. if not how I could get it.

**Comment by xpromache** [comment]
The only thing we have for that is the packet viewer which is a standalone
java (swing) application.
It does not work via the web interface.
